digital-clock/style2.css

```
@@ -0,0 +1,66 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'Quicksand', sans-serif;
+}
+
+.banner{
+    display: flex;
+    align-content: center;
+    justify-content: center;
+    padding: 50px;
+    background-color: #fff;
+}
+.banner-content{
+    padding: 15px;
+    background-color: #fff;
+    box-shadow: -3px -3px 7px #f3f3f3dc, 3px 3px 5px rgba(94, 104, 121, .288);
+    border-radius: 5px;
```

Hows this block suppose to fit-up

---

I added the .profile div and the .time div to the .banner.content div
```suggestion
.banner-content{
    padding: 15px;
    background-color: #fff;
    box-shadow: -3px -3px 7px #f3f3f3dc, 3px 3px 5px rgba(94, 104, 121, .288);
    border-radius: 5px;
```